resources/js/src/app/mixins/url.js

```diff
@@ -418,6 +422,16 @@ export default {
             delete params[Constants.PARAMETER_PAGE];
             delete params[Constants.PARAMETER_ATTRIBUTES];
             document.location.search = '?' + $.param(params);
+        },
+
+        getFacetIdInfoMap() {
```

Comment:
`facetId` looks like something internal of the method to me. I do not think it is crucial for the user of this method, to know how the info map is generated.

Reply:
Not sure if I really understand the problem here. Could you elaborate?

Reply:
I just rechecked the methods return value, and naming it that would be sufficient. I just got confused with id and name of the actual filter.